src/test/util_tests.cpp

```diff
@@ -1239,6 +1239,11 @@ BOOST_AUTO_TEST_CASE(util_FormatMoney)
 BOOST_AUTO_TEST_CASE(util_ParseMoney)
 {
     BOOST_CHECK_EQUAL(ParseMoney("0.0").value(), 0);
+    BOOST_CHECK_EQUAL(ParseMoney(".").value(), 0);
```

Comment:
Maybe it would be worth checking two-dots cases like:

```cpp
ParseMoney("..")
ParseMoney("0..0")
```

From whitespace characters, I can see covered spaces but not tabulators (for example; pattern for `TrimString` is `const std::string& pattern = " \f\n\r\t\v"`). Maybe it would be good to add tests for this too.